sim/firesim-lib/src/main/scala/endpoints/BlockDevWidget.scala

```diff
@@ -77,8 +77,9 @@ class BlockDevWidget(implicit p: Parameters) extends EndpointWidget()(p) {
   rRespBuf.reset  := reset.toBool || targetReset
   wAckBuf.reset  := reset.toBool || targetReset
 
-  io.hPort.toHost.hReady := tFireHelper.fire(io.hPort.toHost.hValid)
-  io.hPort.fromHost.hValid := tFireHelper.fire(io.hPort.fromHost.hReady)
+  // Hack 
```

Comment:
Is it possible to specify in the comment the reason for the hack? (the `fixMeOnNextRocketBump is some indication, but still...)